app/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -121,24 +123,37 @@ public final void setSynapseClient(SynapseClient synapseClient) {
         this.synapseClient = synapseClient;
     }
 
-    public Study getStudy(String identifier) {
+    private Study getStudy(String identifier, boolean includeDeleted) {
         checkArgument(isNotBlank(identifier), Validate.CANNOT_BE_BLANK, "identifier");
-        
+
         Study study = cacheProvider.getStudy(identifier);
         if (study == null) {
             study = studyDao.getStudy(identifier);
             cacheProvider.setStudy(study);
         }
+
+        if (study != null && !study.isActive() && !includeDeleted) {
+            throw new EntityNotFoundException(Study.class, "Study '"+identifier+"' is de-activated.");
```

Comment:
Change message to "Study not found" (this is actually a bit of a security issue, you should not be able to deduce that a study once existed in the system when it has been "deleted").